week-8/exception-handling/src/test/java/com/epam/training/ExceptionHandlingTest.java

```diff
@@ -0,0 +1,24 @@
+package com.epam.training;
+
+import org.junit.Test;
+
+public class ExceptionHandlingTest {
+
+    private final ExceptionHandling underTest = new ExceptionHandling();
+
+    @Test
+    public void testWhenInputIsFloat() {
+        underTest.test(1F);
+    }
+
+    @Test
+    public void testWhenInputIsString() {
+        underTest.test("string");
```

Comment:
I think it would be better to have an executable `main` for this instead of a unit test, maven doesn't behave well when `System.exit` is called.

Reply:
I'm not sure that we would like to build our project with Maven anytime. Also there's one case when the input type is a Float, in that case an unhandled exception will be thrown from the code and our build will fail.
But I think we can remove these test cases, because they don't give us extra benefits (don't assert anything just run the code with the given parameters). What do you think about it?

Reply:
Yes, I think we don't have to go with unit tests for this exercise, especially without any assertion. Maybe with a single entry point (i.e. `main` method) with a try-catch and the proper order of cases or three entry points for different cases would do. Also, `@Test(expected = ParentException.class)` can be used for expected exceptions.

Reply:
I have removed the tests.